pandas-stubs/__init__.pyi

```diff
@@ -32,7 +32,7 @@ def read_csv(
     filepath_or_buffer: Union[str, Path],
     sep: str = ...,
     header: Optional[int] = ...,
-    index_col: Optional[int] = ...,
+    index_col: Optional[Union[str, int, Sequence[str], Sequence[int], bool]] = ...,
```

Comment:
never use `Sequence[str]`

Reply:
Couldn't you have made this a suggestion?